templates/includes/sidebar.html

```diff
@@ -1,6 +1,24 @@
 <ul class="sidebar-nav">
+  
+  {% if user.is_authenticated %}
+  <li>
+    User: <a href="{% url 'taxi:driver-detail' user.pk %}">{{user.username}}</a>     
+  </li>
+  <form action="{% url 'logout' %}" method="post">
+    {% csrf_token %}
+    <button type="submit">Logout</button>
+  </form>
+
+  </li>
```

Comment:
There is an extra closing `</li>` tag here that is not needed. It should be removed to ensure the HTML structure is correct.